core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -543,7 +543,8 @@ def to_dict(self):
         """
         fingerprint_data = {
             unicode(index): {
```

Comment:
We should be calling this id not index right? The two words mean different things -- ID is a unique identifier; index is the position of an element in a list.

Reply:
Done.